iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/ReceiveMessagesTests.java

```diff
@@ -52,7 +45,20 @@ public void setupTest() throws Exception
 
     @Test
     @StandardTierHubOnlyTest
-    public void receiveMessagesOverIncludingProperties() throws Exception
+    public void receiveMessage() throws Exception
+    {
+        receiveMessage(MESSAGE_SIZE_IN_BYTES);
```

Comment:
What does success/failure look like for this test?

Reply:
Presumably the method `receiveMessage` has some asserts in it, and we expect the execution to not throw exceptions?